Structure_Engine/Compute/MapTaperedProfile.cs

```diff
@@ -42,16 +42,19 @@ public static partial class Compute
         /***************************************************/
         [Description("Maps a TaperedProfile to a series of sequential Bars by interpolating the profiles at the startNode and endNode of each bar using a polynomial defined by the interpolationOrder. " +
             "For nonlinear profiles a concave profile is achieved by setting the larger profile at the smallest position. To achieve a convex profile, the larger profile must be at the largest position.")]
-        [Input("section", "The section containing the TaperedProfile to be mapped to the series of Bars")]
+        [Input("section", "The section containing the TaperedProfile to be mapped to the series of Bars.")]
         [Input("bars", "The Bars in sequential order for the TaperedProfile to be mapped to.")]
         [Output("bars", "The Bars with interpolated SectionProperties based on the TaperedProfile provided.")]
         public static List<Bar> MapTaperedProfile(List<Bar> bars, IGeometricalSection section)
         {
+            if (bars.Any(x => x.IsNull("MapTaperedProfile")) || section.IsNull("MapTaperedProfile"))
```

Comment:
If we add the attribute to the IsNull methods in Structure_Engine, we should not have to include the method name here